src/Apps/Order/Routes/Review/index.tsx

```diff
@@ -186,31 +187,69 @@ export class ReviewRoute extends Component<ReviewProps, ReviewState> {
 
     return (
       <>
-        <Row>
-          <Col>
-            <BuyNowStepper currentStep={"review"} />
-          </Col>
-        </Row>
+        <HorizontalPadding px={[0, 4]}>
+          <Row>
+            <Col>
+              <BuyNowStepper currentStep={"review"} />
+            </Col>
+          </Row>
+        </HorizontalPadding>
 
         <Responsive>
           {({ xs }) => (
-            <TwoColumnLayout
-              Content={
-                <>
-                  <Join separator={<Spacer mb={3} />}>
-                    <ShippingAndPaymentReview
-                      order={order}
-                      onChangePayment={this.onChangePayment.bind(this)}
-                      onChangeShipping={this.onChangeShipping.bind(this)}
-                      mb={xs ? 2 : 3}
-                    />
+            <HorizontalPadding px={[2, 4]}>
```

Comment:
I like this double use pattern. Once this is merged I can go back and address my wacky negative margin stuff in `<NavigationTabs>` 😄